addons/point_of_sale/static/src/js/printers.js

```diff
@@ -95,8 +95,11 @@ var PrinterMixin = {
      * Renders the html as an image to print it
      * @param {String} receipt: The receipt to be printed, in HTML
      */
-    htmlToImg: function (receipt) {
+    htmlToImg: function (receipt, is_email = false) {
```

Comment:
You should avoid changing method signature in stable. Maybe you can just create a new method called htmlToImgWithPadding and you put the desired padding on the receiptString and also call the htmlToImg method. It would be more stable